src/crawlFacebook.js

```diff
@@ -0,0 +1,87 @@
+const puppeteer = require('puppeteer');
+const clear = require('clear');
+const chalk = require('chalk');
+const ora = require('ora');
+
+const strings = require('./strings.json');
+const displayEvents = require('./displayEvents');
+
+const crawlFacebook = async (pages, parallel) => {
+	clear();
+
+	const spinner = ora(strings.english.success.crawlingStarted).start();
+	setTimeout(() => {
+		spinner.color = 'blue';
+		spinner.text = strings.english.success.crawlingStarted + '\n';
```

Comment:
Use [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals):

```suggestion
		spinner.text =`${strings.english.success.crawlingStarted}\n`;
```

This is valid for every other string where you concatenate through the app.